src/Algebra/WithApartness/Bundles.agda

```diff
@@ -0,0 +1,54 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Bundles for local algebraic structures
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.WithApartness.Bundles where
+
+open import Level using (_⊔_; suc)
+open import Algebra.Core using (Op₁; Op₂)
+open import Algebra.Bundles using (CommutativeRing)
+open import Algebra.WithApartness.Structures using (IsHeytingCommutativeRing; IsHeytingField)
```

Comment:
I wouldn't bother with a `using` here, as we will by default need everything in this module.

Reply:
agree. removed.